nrcemt/qeels/test/test_peak_detection.py

```diff
@@ -0,0 +1,98 @@
+from nrcemt.qeels.engine.peak_detection import (
+    compute_rect_corners,
+    ycfit,
+    calc_angle,
+    rotate_points
+)
+import math
+import numpy as np
+from scipy.io import loadmat
+
+
+def test_ycfit():
+    # x1:928 y1:273
+    # x2:909 y2:700
+    signal = loadmat('nrcemt/qeels/test/resources/signal.mat')
+    signal_data = signal['Signal']
+    expected_result = loadmat('nrcemt/qeels/test/resources/ycfit.mat')
```

Comment:
using paths in this way relies on the user being in root of the repo when running pytest for the tests to work. I think using a directory relative join like you did in the prz test is best.